### DISCUSSION TITLE
Does Transmissionic support Transmission 4.0.0 beta 1?

### DISCUSSION THREAD

**Discussion by Piddox** [transmission/transmission#4825] [Q&A] [answered]
Does anyone know if Transmissionic supports Transmission 4.0.0 beta 1?
https://github.com/6c65726f79/Transmissionic

I don't wanna update to 4.00 if Transmissionic does not support it. But I would really like to try 4.0.0 as it seems to fix a lot of issues.

**Comment by ckerr** [comment]
It _probably_ will since 4.0.0 didn't have any breaking changes in its RPC API but this is probably a question better asked in the Transmissionic forums for https://github.com/6c65726f79/Transmissionic/discussions or https://github.com/6c65726f79/Transmissionic/issues

CC @6c65726f79

**Comment by 6c65726f79** [comment, marked as answer]
I can confirm, it works just fine.
